contributing/community/reviews.rst

```diff
@@ -0,0 +1,190 @@
+  Is any important information missing? Can the bug easily be reproduced?
+
+* **Pull Requests**: Pull requests contain code that fixes a bug or implements
+  new functionality. Reviews of pull requests ensure that they are implemented
+  properly, are covered by test cases, don't introduce new bugs and maintain
+  backwards compatibility.
+
+Note that **anyone who has some basic familiarity with Symfony and PHP can
+review bug reports and pull requests**. You don't need to be an expert to help.
+
+Be Constructive
+---------------
+
+Before you begin, remember that you are looking at the result of someone else's
+hard work. A good review post thanks the contributor for their work, identifies
+what was done well, identifies what should be improved, and suggests a next
+step.
+
+Create a GitHub Account
+-----------------------
+
+Symfony uses GitHub_ to manage bug reports and pull requests. If you want to
+do reviews, you need to `create a GitHub account`_ and log in.
+
+The Bug Report Review Process
+-----------------------------
+
+A good way to get started with reviewing is to pick a bug report from the
+`bug reports in need of review`_.
+
+The steps for the review are:
+
+1. **Is the Report Complete?**
+
+   Good bug reports contain a link to a fork of the `Symfony Standard Edition`_
+   (the "reproduction project") that reproduces the bug. If it doesn't, the
+   report should at least contain enough information and code samples to
+   reproduce the bug.
+
+2. **Reproduce the Bug**
+
+   Download the reproduction project and test whether the bug can be reproduced
+   on your system. If the reporter did not provide a reproduction project,
+   create one by forking_ the `Symfony Standard Edition`_. Reproduce the bug
+   with the instructions given by the reporter.
+
+3. **Update the Issue Status**
+
+   At last, add a comment and update the status of the bug report. **Thank the
+   reporter for reporting the bug**. Include the line ``Status: <status>`` in
+   your comment to update the status to one of the following:
+
+   **Needs Work** If the bug does not contain enough information to be
+   reproduced, explain what information is missing and move the report to this
+   status.
+
+   **Works for me** If the bug *does* contain enough information to be
+   reproduced but works on your system, or if the reported bug is a feature and
+   not a bug, provide a short explanation and move the report to this status.
+
+   **Reviewed** If you can reproduce the bug, move the report to this status.
+   If you created a reproduction project, include the link to the project in
+   your comment.
+
+.. topic:: Example
+
+    Here is a sample comment for a bug report that could be reproduced:
+
+    .. code-block:: text
+
+        Thank you @weaverryan for creating this bug report! This indeed looks
+        like a bug. I reproduced the bug in the "kernel-bug" branch of
+        https://github.com/webmozart/symfony-standard.
```

Comment:
should'nt the link just be ``https://github.com/webmozart/symfony-standard/tree/kernel-bug`` ?